baby-gru/src/components/card/MoorhenMoleculeRepresentationSettingsCard.tsx

```diff
@@ -69,6 +70,14 @@ export const MoorhenMoleculeRepresentationSettingsCard = (props: {
         }
     }, [symmetryOn])
 
+    useEffect(() => {
+        if (props.molecule.biomolOn !== biomolOn) {
+            if(biomolOn)
+                props.molecule.symmetryOn = false
```

Comment:
It seems that `molecule.symmetryOn = false` is already happening inside `molecule.toggleBiomolcule` in which case this here is unnecessary?

Reply:
Done in 98e02208e3be2dfe1892e1d9bf9ac019e067b9d4.